.github/workflows/zxf-publish-docker-images.yaml

```diff
@@ -0,0 +1,261 @@
+    runs-on: [ self-hosted, Linux, large, ephemeral ]
+    needs:
+      - prepare-release
+      - deploy-base-image
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # pinned to v3.3.0
+
+      - name: Authenticate to Google Cloud
+        id: google-auth
+        uses: google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d # pinned to v1.0.0
+        with:
+          workload_identity_provider: 'projects/101730247931/locations/global/workloadIdentityPools/hedera-registry-pool/providers/hedera-registry-gh-actions'
+          service_account: 'swirlds-automation@hedera-registry.iam.gserviceaccount.com'
+          token_format: access_token
+
+      - name: Setup Docker Buildx QEMU
+        uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18 # pinned to v2.1.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@f03ac48505955848960e80bbb68046aa35c7b9e7 # pinned to v2.4.1
+
+      - name: Authorize Docker
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # pinned to v2.1.0
+        with:
+          registry: gcr.io
+          username: oauth2accesstoken
+          password: ${{ steps.google-auth.outputs.access_token }}
+
+      - name: Download Artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # pinned to v3.0.2
+        with:
+          name: hedera-node-build-${{ needs.prepare-release.outputs.tag }}
+          path: docker/main-network-node/sdk
+
+      - name: Extract Artifact Contents
+        working-directory: docker/main-network-node/sdk
+        run: unzip "build-${{ needs.prepare-release.outputs.tag }}.zip" "data/*"
+
+      - name: Build and Push Image
+        uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671 # pinned to v4.0.0
+        with:
+          push: true
+          context: ${{ github.workspace }}/docker/main-network-node
+          platforms: |
+            linux/arm64
+            linux/amd64
+          tags: |
+            gcr.io/hedera-registry/main-network-node:${{ needs.prepare-release.outputs.version }}
+          build-args: |
+            IMAGE_TAG=${{ needs.prepare-release.outputs.version }}
+
+  report-published-images:
+    name: Deploy / Base / Image
+    runs-on: [ self-hosted, Linux, large, ephemeral ]
+    needs:
+      - prepare-release
+      - deploy-base-image
+      - deploy-network-node-image
+      - deploy-haveged-image
+    steps:
+      - name: Render Job Summary
+        run: |
+          printf "### Published Docker Images\n" >> "${GITHUB_STEP_SUMMARY}"
+          printf "| Image Name | Version | URL | Supported Architectures |\n" >> "${GITHUB_STEP_SUMMARY}"
+          printf "| ---------- | ------- | --- | ----------------------- |\n" >> "${GITHUB_STEP_SUMMARY}"
+          printf "| %s | %s | %s | %s |\n" \
+            "gcr.io/hedera-regsitry/network-node-base" \
```

Comment:
typo - should be `registry`, image is unreachable

Reply:
Fixed